packages/openapi3/src/openapi.ts

```diff
@@ -247,18 +247,14 @@ function createOAPIEmitter(program: Program, options: OpenAPIEmitterOptions) {
     }
     const versions = getVersionRecords(program, serviceNs);
     for (const record of versions) {
-      record.projections.push({
```

Comment:
@bterlson was this meant to be here. It causes some issue with the change below with `enableProjections` always running

Reply:
I believe that the toTypescript projection is currently the default projection that all emitters must run.  The design issue on side cars (#218) calls for clarifying a standard set of projections and which are supposed to run.

Reply:
Fine with removing this. It somewhat helpful for demos but it was mistakenly adding it so removing it is fine.